error/error.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Implementación de la clase de Error
+ */
+
+export default class Error 
```

Comment:
ya hay un [`Error`](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Error). Si creáis uno, que sea específico para vuestro problema. Es más, hay muchos errores más concretos que podéis extender  para vuestro problema.